### PR TITLE
Fix #5739 Remove distribution from doco

### LINKED ISSUE #5739: jetty-distribution is not created when building 11.0.0.beta3
**Jetty version** - 11.0.0.beta3
**Java version** - 11
**OS type/version** - Debian 9
**Description**  - jetty-distribution folder is not created 

$git log
```
Author: Joakim Erdfelt <joakim.erdfelt@gmail.com>
Date:   Wed Oct 21 16:06:13 2020 -0500

    Updating to version 11.0.0.beta3
```
This is output:

```
[INFO] Reactor Summary for Jetty :: Project 11.0.0.beta3:
[INFO] 
[INFO] Jetty :: Build Resources ........................... SUCCESS [  0.909 s]
[INFO] Jetty :: Project ................................... SUCCESS [  3.259 s]
[INFO] Jetty :: Slf4j Implementation ...................... SUCCESS [  5.187 s]
[INFO] Jetty :: Utilities ................................. SUCCESS [ 11.362 s]
[INFO] Jetty :: IO Utility ................................ SUCCESS [  2.255 s]
[INFO] Jetty :: Http Utility .............................. SUCCESS [  4.464 s]
[INFO] Jetty :: ALPN :: Parent ............................ SUCCESS [  0.088 s]
[INFO] Jetty :: ALPN :: Client ............................ SUCCESS [  0.369 s]
[INFO] Jetty :: JMX Management ............................ SUCCESS [  1.228 s]
[INFO] Jetty :: XML utilities ............................. SUCCESS [  1.712 s]
[INFO] Jetty Tests :: Parent .............................. SUCCESS [  0.063 s]
[INFO] Jetty Tests :: HTTP Utilities ...................... SUCCESS [  0.351 s]
[INFO] Jetty :: Utilities :: Ajax(JSON) ................... SUCCESS [  1.167 s]
[INFO] Jetty :: Server Core ............................... SUCCESS [ 13.180 s]
[INFO] Jetty :: Security .................................. SUCCESS [  1.863 s]
[INFO] Jetty :: Asynchronous HTTP Client .................. SUCCESS [  5.972 s]
[INFO] Demos :: Parent .................................... SUCCESS [  0.090 s]
[INFO] Demo :: Async Rest ................................. SUCCESS [  0.061 s]
[INFO] Demo :: Async Rest :: Jar .......................... SUCCESS [  0.381 s]
[INFO] Demo :: Async Rest :: WebApp ....................... SUCCESS [  0.337 s]
[INFO] Jetty :: Servlet Handling .......................... SUCCESS [  3.233 s]
[INFO] Jetty :: Webapp Application Support ................ SUCCESS [  2.229 s]
[INFO] Demo :: Async Rest :: Server ....................... SUCCESS [  0.252 s]
[INFO] Demo :: JAAS :: WebApp ............................. SUCCESS [  0.150 s]
[INFO] Demo :: Mock Resources ............................. SUCCESS [  0.250 s]
[INFO] Demo :: JNDI :: WebApp ............................. SUCCESS [  0.516 s]
[INFO] Jetty :: Utility Servlets and Filters .............. SUCCESS [  1.956 s]
[INFO] Jetty :: Websocket :: Parent ....................... SUCCESS [  0.079 s]
[INFO] Jetty :: Websocket :: org.eclipse.jetty.websocket :: API SUCCESS [  0.496 s]
[INFO] Jetty :: Websocket :: Core :: Common ............... SUCCESS [  1.751 s]
[INFO] Jetty :: Websocket :: org.eclipse.jetty.websocket :: Util SUCCESS [  0.894 s]
[INFO] Jetty :: Websocket :: org.eclipse.jetty.websocket :: Common SUCCESS [  1.058 s]
[INFO] Jetty :: Websocket :: Core :: Server ............... SUCCESS [  0.749 s]
[INFO] Jetty :: Websocket :: org.eclipse.jetty.websocket :: Util-Server SUCCESS [  0.541 s]
[INFO] Jetty :: JNDI Naming ............................... SUCCESS [  0.783 s]
[INFO] Jetty :: Plus ...................................... SUCCESS [  1.018 s]
[INFO] Jetty :: Servlet Annotations ....................... SUCCESS [  1.415 s]
[INFO] Jetty :: Websocket :: org.eclipse.jetty.websocket :: Server SUCCESS [  0.571 s]
[INFO] Jetty :: Websocket :: Core :: Client ............... SUCCESS [  0.505 s]
[INFO] Jetty :: Websocket :: jakarta.websocket :: Common .. SUCCESS [  1.202 s]
[INFO] Jetty :: Websocket :: jakarta.websocket :: Client .. SUCCESS [  0.476 s]
[INFO] Jetty :: Websocket :: jakarta.websocket :: Server .. SUCCESS [  0.645 s]
[INFO] Demo :: Jetty :: WebApp ............................ SUCCESS [  1.017 s]
[INFO] Jetty :: Proxy ..................................... SUCCESS [  1.634 s]
[INFO] Jetty :: HTTP2 ..................................... SUCCESS [  0.057 s]
[INFO] Jetty :: HTTP2 :: HPACK ............................ SUCCESS [  0.852 s]
[INFO] Jetty :: HTTP2 :: Common ........................... SUCCESS [  1.646 s]
[INFO] Jetty :: ALPN :: Server ............................ SUCCESS [  0.285 s]
[INFO] Jetty :: HTTP2 :: Server ........................... SUCCESS [  0.866 s]
[INFO] Jetty :: ALPN :: JDK9 Server Implementation ........ SUCCESS [  0.297 s]
[INFO] Demo :: Proxy :: Webapp ............................ SUCCESS [  0.243 s]
[INFO] Demo :: Servlet Spec :: Fragment Jar ............... SUCCESS [  0.174 s]
[INFO] Demo :: Servlet Spec :: ServletContainerInitializer Jar SUCCESS [  0.205 s]
[INFO] Demo :: Servlet Spec :: Webapp ..................... SUCCESS [  0.512 s]
[INFO] Demo :: Servlet Spec ............................... SUCCESS [  0.065 s]
[INFO] Jetty :: Deployers ................................. SUCCESS [  0.825 s]
[INFO] Jetty :: Rewrite Handler ........................... SUCCESS [  0.755 s]
[INFO] Jetty :: HTTP2 :: Client ........................... SUCCESS [  1.500 s]
[INFO] Jetty :: ALPN :: Conscrypt Client Implementation ... SUCCESS [  0.424 s]
[INFO] Jetty :: ALPN :: JDK9 Client Implementation ........ SUCCESS [  0.252 s]
[INFO] Jetty :: HTTP2 :: HTTP Client Transport ............ SUCCESS [  0.739 s]
[INFO] Jetty :: ALPN :: Conscrypt Server Implementation ... SUCCESS [  0.324 s]
[INFO] Jetty :: JAAS ...................................... SUCCESS [  1.921 s]
[INFO] Jetty :: Apache JSP Implementation ................. SUCCESS [  0.456 s]
[INFO] Apache :: JSTL module .............................. SUCCESS [  0.301 s]
[INFO] Jetty :: Websocket :: org.eclipse.jetty.websocket :: Client SUCCESS [  0.710 s]
[INFO] Jetty :: Quick Start ............................... SUCCESS [  0.989 s]
[INFO] Jetty :: Start ..................................... SUCCESS [  1.875 s]
[INFO] Jetty :: UnixSocket ................................ SUCCESS [  0.047 s]
[INFO] Jetty :: UnixSocket :: Common ...................... SUCCESS [  0.279 s]
[INFO] Jetty :: UnixSocket :: Server ...................... SUCCESS [  0.583 s]
[INFO] Jetty :: FastCGI :: Parent ......................... SUCCESS [  0.045 s]
[INFO] Jetty :: FastCGI :: Client ......................... SUCCESS [  0.678 s]
[INFO] Jetty :: FastCGI :: Server ......................... SUCCESS [  0.795 s]
[INFO] Example :: Jetty Spring ............................ SUCCESS [  0.367 s]
[INFO] Jetty :: CDI ....................................... SUCCESS [  0.453 s]
[INFO] Jetty :: OpenID .................................... SUCCESS [  0.723 s]
[INFO] Jetty :: JASPI Security ............................ SUCCESS [  0.547 s]
[INFO] Jetty :: Infinispan ................................ SUCCESS [  0.044 s]
[INFO] Jetty :: Infinispan Session Manager Common ......... SUCCESS [  0.700 s]
[INFO] Jetty :: Infinispan Session Manager Embedded ....... SUCCESS [  0.145 s]
[INFO] Jetty :: Infinispan Session Manager Embedded with Querying SUCCESS [  0.787 s]
[INFO] Jetty :: Infinispan Session Manager Remote ......... SUCCESS [  0.286 s]
[INFO] Jetty :: Infinispan Session Manager Remote with Querying SUCCESS [  0.837 s]
[INFO] Jetty :: Hazelcast Session Manager ................. SUCCESS [  0.729 s]
[INFO] Jetty :: GCloud .................................... SUCCESS [  0.087 s]
[INFO] Jetty :: GCloud :: Session Manager ................. SUCCESS [  0.908 s]
[INFO] Jetty :: Memcached ................................. SUCCESS [  0.060 s]
[INFO] Jetty :: Memcached :: Sessions ..................... SUCCESS [  0.409 s]
[INFO] Jetty Tests :: Sessions :: Parent .................. SUCCESS [  0.058 s]
[INFO] Jetty Tests :: Sessions :: Common .................. SUCCESS [  1.166 s]
[INFO] Jetty :: NoSQL Session Managers .................... SUCCESS [  0.543 s]
[INFO] Jetty :: Home Assembly ............................. SUCCESS [  2.218 s]
[INFO] Demo :: Embedded Jetty ............................. SUCCESS [  1.542 s]
[INFO] Jetty :: Ant Plugin ................................ SUCCESS [  0.591 s]
[INFO] Jetty :: Websocket :: Core :: Tests ................ SUCCESS [  1.430 s]
[INFO] Jetty :: Websocket :: org.eclipse.jetty.websocket :: Tests SUCCESS [  0.861 s]
[INFO] Jetty :: Websocket :: jakarta.websocket :: Tests ... SUCCESS [  2.629 s]
[INFO] Jetty :: Jetty Maven Plugin ........................ SUCCESS [  5.171 s]
[INFO] Jetty :: Jetty JSPC Maven Plugin ................... SUCCESS [  1.948 s]
[INFO] Jetty :: UnixSocket :: Client ...................... SUCCESS [  0.348 s]
[INFO] Jetty :: Jmh ....................................... SUCCESS [  3.896 s]
[INFO] Jetty Tests :: WebApps :: Parent ................... SUCCESS [  0.043 s]
[INFO] Jetty Tests :: WebApp :: RFC2616 ................... SUCCESS [  0.171 s]
[INFO] Test :: Jetty HTTP2 Webapp ......................... SUCCESS [  0.355 s]
[INFO] Test :: Jetty Test Simple Webapp ................... SUCCESS [  0.069 s]
[INFO] Test :: Jetty Felix Webapp ......................... SUCCESS [  0.184 s]
[INFO] Test :: CDI :: Common Demo Webapp .................. SUCCESS [  0.151 s]
[INFO] Test :: CDI On Jetty :: Weld Demo Webapp ........... SUCCESS [  0.276 s]
[INFO] Test :: CDI On Jetty :: OWB Demo Webapp ............ SUCCESS [  0.212 s]
[INFO] Test :: Jetty Websocket Simple Webapp .............. SUCCESS [  0.139 s]
[INFO] Test :: Jetty Bad Websocket Simple Webapp .......... SUCCESS [  0.134 s]
[INFO] Jetty Tests :: Sessions :: File .................... SUCCESS [  0.207 s]
[INFO] Jetty Tests :: Sessions :: JDBC .................... SUCCESS [  0.253 s]
[INFO] Jetty Tests :: Sessions :: Mongo ................... SUCCESS [  0.252 s]
[INFO] Jetty Tests :: Sessions :: Infinispan .............. SUCCESS [  0.469 s]
[INFO] Jetty Tests :: Sessions :: GCloud .................. SUCCESS [  0.376 s]
[INFO] Jetty Tests :: Sessions :: Memcached ............... SUCCESS [  0.189 s]
[INFO] Jetty Tests :: Sessions :: Hazelcast ............... SUCCESS [  0.305 s]
[INFO] Jetty Tests :: Login Service ....................... SUCCESS [  0.213 s]
[INFO] Jetty Tests :: Integrations ........................ SUCCESS [  1.122 s]
[INFO] Test :: Jetty Quick Start .......................... SUCCESS [  0.349 s]
[INFO] Jetty Tests :: JMX Parent .......................... SUCCESS [  0.047 s]
[INFO] Jetty Tests :: JMX :: WebApp ....................... SUCCESS [  0.162 s]
[INFO] Jetty Tests :: JMX :: WebApp Integration Tests ..... SUCCESS [  0.164 s]
[INFO] Test :: HTTP Client Transports ..................... SUCCESS [  1.137 s]
[INFO] test-distribution .................................. SUCCESS [  0.786 s]
[INFO] Test :: CDI ........................................ SUCCESS [  0.338 s]
[INFO] Jetty :: Runner .................................... SUCCESS [  3.042 s]
[INFO] Jetty :: Http Service Provider Interface ........... SUCCESS [  0.547 s]
[INFO] Jetty :: OSGi ...................................... SUCCESS [  0.049 s]
[INFO] Jetty :: OSGi :: Boot .............................. SUCCESS [  0.835 s]
[INFO] Jetty :: OSGi :: Boot JSP .......................... SUCCESS [  0.341 s]
[INFO] Jetty :: OSGi :: Boot :: Warurl .................... SUCCESS [  0.313 s]
[INFO] Jetty :: OSGi :: HttpService ....................... SUCCESS [  0.199 s]
[INFO] Jetty :: OSGi :: Test WebApp ....................... SUCCESS [  0.233 s]
[INFO] OSGi Test :: Webapp With Resources ................. SUCCESS [  0.164 s]
[INFO] Jetty :: OSGi :: Test Context ...................... SUCCESS [  0.249 s]
[INFO] Jetty :: OSGi :: WebApp Fragment ................... SUCCESS [  0.107 s]
[INFO] Jetty :: OSGi :: Server ............................ SUCCESS [  0.216 s]
[INFO] Jetty :: OSGi ALPN Fragment ........................ SUCCESS [  0.099 s]
[INFO] Jetty :: OSGi :: Test .............................. SUCCESS [  1.236 s]
[INFO] Jetty :: Bom ....................................... SUCCESS [  0.206 s]
[INFO] Jetty :: Documentation ............................. SUCCESS [ 22.139 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS

$ls -1
advisories
apache-jsp
apache-jstl
build-resources
CODE_COVERAGE.md
CONTRIBUTING.md
demos
header-template.txt
javadoc
Jenkinsfile
jetty-alpn
jetty-annotations
jetty-ant
jetty-bom
jetty-cdi
jetty-client
jetty-deploy
jetty-documentation
jetty-fcgi
jetty-gcloud
jetty-hazelcast
jetty-home
jetty-http
jetty-http2
jetty-http-spi
jetty-infinispan
jetty-io
jetty-jaas
jetty-jaspi
jetty-jmh
jetty-jmx
jetty-jndi
jetty-jspc-maven-plugin
jetty-maven-plugin
jetty-memcached
jetty-nosql
jetty-openid
jetty-osgi
jetty-plus
jetty-proxy
jetty-quickstart
jetty-rewrite
jetty-runner
jetty-security
jetty-server
jetty-servlet
jetty-servlets
jetty-slf4j-impl
jetty-spring
jetty-start
jetty-unixsocket
jetty-util
jetty-util-ajax
jetty-webapp
jetty-websocket
jetty-xml
Jmh_Jenkinsfile
KEYS.txt
LICENSE
logos
NOTICE.txt
pom.xml
README.md
scripts
target
tests
VERSION.txt

```

### PR DESCRIPTION
**Pull request by gregw**
Fix #5739 Remove distribution from doco
Mostly mechanical fixes

### CODE REVIEW

**Comment by gregw** [comment]
I don't think we should delay to make this perfect, specially in the old_docs.
We need to make it not wrong as much as possible and continue to update docs after this PR is merged

File: jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc [hunk 1]
```diff
@@ -298,7 +298,7 @@ org.eclipse.jetty.server.Server@76f08fe1 - STARTING
  |   |   |   |
  |   |   |   +> No ClassLoader
  |   |   |   +> org.eclipse.jetty.server.webapp.ContainerIncludeJarPattern=.*/jetty-servlet-api-[^/]*\.jar$
- |   |   += o.e.j.w.WebAppContext@6f01ba6f{/,file:/home/user/jetty-distribution-{VERSION}/webapps/ROOT/,AVAILABLE}{/ROOT} - STARTED
+ |   |   += o.e.j.w.WebAppContext@6f01ba6f{/,file:/home/user/jetty-home-{VERSION}/webapps/ROOT/,AVAILABLE}{/ROOT} - STARTED
```
Comment (joakime) [inline jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc]: This shouldn't be jetty-home.../webapps, should it?

File: jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc [hunk 2]
```diff
@@ -320,7 +320,7 @@ org.eclipse.jetty.server.Server@76f08fe1 - STARTING
...
...jdk/Contents/Home/jre/lib/ext/sunec.jar:/Library/Java/JavaVirtualMachines/jdk1.7.0_17.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar:/Library/Java/JavaVirtualMachines/jdk1.7.0_17.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar:/Library/Java/JavaVirtualMachines/jdk1.7.0_17.jdk/Contents/Home/jre/lib/ext/zipfs.jar:/opt/local/lib/libsvnjavahl-1.0.dylib:/System/Library/Java/Extensions/AppleScriptEngine.jar:/System/Library/Java/Extensions/dns_sd.jar:/System/Library/Java/Extensions/j3daudio.jar:/System/Library/Java/Extensions/j3dcore.jar:/System/Library/Java/Extensions/j3dutils.jar:/System/Library/Java/Extensions/jai_codec.jar:/System/Library/Java/Extensions/jai_core.jar:/System/Library/Java/Extensions/libAppleScriptEngine.jnilib:/System/Library/Java/Extensions/libJ3D.jnilib:/System/Library/Java/Extensions/libJ3DAudio.jnilib:/System/Library/Java/Extensions/libJ3DUtils.jnilib:/System/Library/Java/Extensions/libmlib_jai.jnilib:/System/Library/Java/Extensions/libQTJNative.jnilib:/System/Library/Java/Extensions/mlibwrapper_jai.jar:/System/Library/Java/Extensions/MRJToolkit.jar:/System/Library/Java/Extensions/QTJava.zip:/System/Library/Java/Extensions/vecmath.jar:/usr/lib/java/libjdns_sd.jnilib
```
Comment (joakime) [inline jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc]: We should regen this, it still references Java 1.7.0_17

File: jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc [hunk 3]
```diff
@@ -342,39 +342,39 @@ org.eclipse.jetty.server.Server@76f08fe1 - STARTING
...
  |   |   |   |       +- sun.misc.Launcher$AppClassLoader@19d1b44b
- |   |   |   |           +- file:/home/user/jetty-distribution-{VERSION}/start.jar
+ |   |   |   |           +- file:/home/user/jetty-home-{VERSION}/start.jar
  |   |   |   |           +- sun.misc.Launcher$ExtClassLoader@1693b52b
  |   |   |   +> javax.servlet.context.tempdir=/private/var/folders/br/kbs2g3753c54wmv4j31pnw5r0000gn/T/jetty-0.0.0.0-9090-ROOT-_-any-
  |   |   |   +> org.eclipse.jetty.server.webapp.ContainerIncludeJarPattern=.*/jetty-servlet-api-[^/]*\.jar$
  |   |   |   +> com.sun.jsp.taglibraryCache={}
  |   |   |   +> com.sun.jsp.tagFileJarUrlsCache={}
- |   |   += o.e.j.s.h.ContextHandler@7b2dffdf{/javadoc,file:/home/user/jetty-distribution-{VERSION}/javadoc,AVAILABLE} - STARTED
+ |   |   += o.e.j.s.h.ContextHandler@7b2dffdf{/javadoc,file:/home/user/jetty-home-{VERSION}/javadoc,AVAILABLE} - STARTED
```
Comment (joakime) [inline jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc]: This `/javadoc` webapp doesn't exist

File: jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc [hunk 4]
```diff
@@ -403,7 +403,7 @@ org.eclipse.jetty.server.Server@76f08fe1 - STARTING
...
...jdk/Contents/Home/jre/lib/ext/sunec.jar:/Library/Java/JavaVirtualMachines/jdk1.7.0_17.jdk/Contents/Home/jre/lib/ext/sunjce_provider.jar:/Library/Java/JavaVirtualMachines/jdk1.7.0_17.jdk/Contents/Home/jre/lib/ext/sunpkcs11.jar:/Library/Java/JavaVirtualMachines/jdk1.7.0_17.jdk/Contents/Home/jre/lib/ext/zipfs.jar:/opt/local/lib/libsvnjavahl-1.0.dylib:/System/Library/Java/Extensions/AppleScriptEngine.jar:/System/Library/Java/Extensions/dns_sd.jar:/System/Library/Java/Extensions/j3daudio.jar:/System/Library/Java/Extensions/j3dcore.jar:/System/Library/Java/Extensions/j3dutils.jar:/System/Library/Java/Extensions/jai_codec.jar:/System/Library/Java/Extensions/jai_core.jar:/System/Library/Java/Extensions/libAppleScriptEngine.jnilib:/System/Library/Java/Extensions/libJ3D.jnilib:/System/Library/Java/Extensions/libJ3DAudio.jnilib:/System/Library/Java/Extensions/libJ3DUtils.jnilib:/System/Library/Java/Extensions/libmlib_jai.jnilib:/System/Library/Java/Extensions/libQTJNative.jnilib:/System/Library/Java/Extensions/mlibwrapper_jai.jar:/System/Library/Java/Extensions/MRJToolkit.jar:/System/Library/Java/Extensions/QTJava.zip:/System/Library/Java/Extensions/vecmath.jar:/usr/lib/java/libjdns_sd.jnilib
```
Comment (joakime) [inline jetty-documentation/src/main/asciidoc/old_docs/logging/dump-tool.adoc]: Java 1.7 again.

**Review by joakime — CHANGES_REQUESTED** [review]
Need a few updates.
I'll submit a few followup commits with them.

**Review by joakime — APPROVED** [review]
I'm good with the changes as they currently stand.

**Comment by joakime** [comment]
@gregw once we get a green build from CI, I'll merge this.

**Pull request merged**